chat/indico_chat/templates/manage_event.html

```diff
@@ -27,7 +27,14 @@
                                         {% if logs_enabled and not chatroom.custom_server %}
                                             <a href="{{ url_for_plugin('.manage_rooms_logs', event_chatroom) }}" class="i-button icon-file-content" title="{% trans %}Logs{% endtrans %}"></a>
                                         {% endif %}
-                                        <a href="{{ url_for_plugin('.manage_rooms_modify', event_chatroom) }}" class="i-button icon-edit" title="{%- trans %}Edit{% endtrans -%}"></a>
+                                        <a href="#" class="i-button icon-edit"
+                                           data-href="{{ url_for_plugin('.manage_rooms_modify', event_chatroom) }}"
+                                           data-title="{%- trans %}Edit chatroom{% endtrans -%}"
+                                           data-ajax-dialog
+                                           data-hide-page-header
```

Comment:
this is not needed if you adapt the template properly (remove inheritance and the blocks)